t/pipe.t

```diff
@@ -210,6 +210,6 @@ exited
     }
 --- response_body
 ok
---- shutdown_error_log
+--- error_log
```

Comment:
Why do you change this?
The `--- shutdown_error_log` section is used for checking the error logs generated during nginx exit.
The `--- error_log` to test normal error logs
I think they are for different purpose

Reply:
I found that log appears in `error_log`, but not in `shutdown_error_log`. also confirmed this by modifying the source code of test-nginx.

Reply:
we need to merge this PR first.

this change seems acceptable

@monkeyDluffy6017 if you have different though, you can create new issue or PR to fix it